pandasdmx/message.py

```diff
@@ -92,6 +93,26 @@ class Config:
     #: that returned the Message. This is not part of the SDMX standard.
     response: Response = None
 
+    def to_pandas(self, *args, **kwargs):
+        '''
+        Convert a :inst:`Message` to :mod:`pandas` object(s) by
```

Comment:
- `:inst:` is not a [Sphinx ReST role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html). 
- Within a class's docstring or the docstring of its methods, it is not necessary to use any special markup to refer to the class. See below where 'DataMessage' is used without markup in the docstring of that class.